jury/metrics/accuracy/accuracy_for_language_generation.py

```diff
@@ -104,7 +108,11 @@ def _compute_single_pred_single_ref(
             for token, pred_count in pred_counts.items():
                 if token in ref_counts:
                     score += min(pred_count, ref_counts[token])  # Intersection count
```

Comment:
This part is on me, and it can be refactored on this pr, can you please refactor the part L105-110 as follows

```python
for pred, ref in zip(predictions, references):
    common = Counter(pred) & Counter(ref)  # Intersection count
    score = sum(common.values())
```

We put this nicer way on precision and recall metrics, but it seems we forgot to add that in accuracy.

Reply:
Working on it.